db/migrate/20180719162241_create_amr_counts.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAmrCounts < ActiveRecord::Migration[5.1]
+  def change
+    create_table :amr_counts do |t|
+      t.string :sample_id
```

Comment:
Since you say amr_count belongs to pipeline_run, you need to have a column pipeline_run_id here (bigint). Also, remove sample_id. If I rerun the pipeline for a sample (new pipeline_run), the resulting amr_counts should be associated with that new pipeline_run, and the old pipeline_run would still have its own amr_counts. So the relationship is on the pipeline_run level, not the sample level.

Reply:
Ok, saw you have pipeline_run_id. But sample_id still needs to be removed.